engines/winnative/payload-schema.yml

```diff
@@ -0,0 +1,57 @@
+id: "http://schemas.taskcluster.net/generic-worker/v1/payload.json#"
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Payload"
+description: "This schema defines the structure of the `payload` property referred to in a Task Cluster Task definition."
+type: "object"
+required: 
+  - "command"
+  - "maxRunTime"
+properties: 
+  command: 
+    title: "Commands to run"
+    type: "array"
+    minItems: 1
+    items: 
+      type: "array"
+      minItems: 1
+      items: 
+        type: "string"
+    description: "One array per command (each command is an array of arguments). Several arrays for several commands."
```

Comment:
In linux, it is relatively straightforward to exit-on-error, e.g. in bash with `#!/bin/bash -e` or `set -e`. On Windows, there is no equivalent, so for most tasks, which will be a sequence of commands, it will be non-trivial for tasks to exit if one of the steps fail. Typically you also need at least two steps: one to checkout sources, and the second step to execute a script. I'm not sure if there is an equivalent construct to `&&` for batch scripts.